great-idea-website/index.html

```diff
@@ -16,48 +16,127 @@
 
   <body>
 
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
 
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
 
-  Get Started
+  <header>
+    <div class="nav">
+    <nav>
+    <a class="active" href="#services">Services</a>
+    <a href="#product">Product</a>
+    <a href="#vision">Vision</a>
+    <a href="#features">Features</a>
+    <a href="#about">About</a>
+    <a href="#contact">Contact</a>
+  </nav>
+</div>
+    
```

Comment:
Your indentations are off so it makes it harder to read your code. Try building a habit of prettifying your code. There is also an extension called Prettier in VS Code that can assist you with that. https://github.com/prettier/prettier